src/styles/components/typography/styles.less

```diff
@@ -114,10 +114,6 @@
     color: @link-color;
   }
 
-  .text-small {
-    font-size: 0.7rem;
```

Comment:
.small in CNVS is actually a bit bigger `@small-font-size: 0.8rem;` are you fine with having it a bit bigger in this place?